migrations/mysql/20240603102429_access-tokens_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE access_tokens (
+    id CHAR(36) DEFAULT (UUID()),
```

Comment:
I don't think we need this default, since this is backend-defined identity case.

Reply:
totally agree, removed